openstack/ces/v1/metricdata/requests.go

```diff
@@ -4,141 +4,127 @@ import (
 	"github.com/opentelekomcloud/gophertelekomcloud"
 )
 
-// BatchQueryOptsBuilder allows extensions to add additional parameters to the
-// BatchQuery request.
-type BatchQueryOptsBuilder interface {
-	ToBatchQueryOptsMap() (map[string]interface{}, error)
-}
-
-type Metric struct {
-	// Specifies the namespace in service.
-	Namespace string `json:"namespace" required:"true"`
-
-	// The value can be a string of 1 to 64 characters
-	// and must start with a letter and contain only uppercase
-	// letters, lowercase letters, digits, and underscores.
-	MetricName string `json:"metric_name" required:"true"`
-
-	// Specifies the list of the metric dimensions.
-	Dimensions []Dimension `json:"dimensions" required:"true"`
-}
-
-// BatchQueryOpts represents options for batch query metric data.
-type BatchQueryOpts struct {
-	// Specifies the metric data.
+type BatchListMetricDataRequest struct {
```

Comment:
As discussed in https://github.com/opentelekomcloud/gophertelekomcloud/issues/274, builder should stay